content/es-us/docs/latest/ASS_Tags.md

```diff
@@ -0,0 +1,816 @@
+1. Centro izquierda
+1. Centro
+1. Centro derecha
+1. Arriba izquierda
+1. Arriba centro
+1. Arriba derecha
+
+{{<tag-def-box title="Alineación de línea (antigua)" id="\a">}}\\a<i>\<pos></i>{{</tag-def-box>}}
+Especifica la alineación de la línea utilizando códigos de alineación heredados de SubStation Alpha. Esta etiqueta es compatible pero se considera obsoleta; normalmente uno debe usar `\an` en nuevos guiones, ya que es más intuitivo.
+
+La excepción es que `\a6` debería usarse para la traducción perezosa de signos, porque si uno va a ser holgazán ha de hacerlo bien y ahorrar la letra extra.
+
+Calcule _pos_ de la siguiente manera: use 1 para alineación a la izquierda, 2 para alineación central y 3 para alineación a la derecha. Si quiere títulos abajo, ya está. Para títulos arriba, agregue 4 al número, y para títulos intermedios agregue 8 al número:
+
+- 1: Bajo izquierda
+- 2: Bajo centro
+- 3: Bajo derecha
+- 5: Arriba izquierda
+- 6: Arriba centro
+- 7: Arriba derecha
+- 9: Centro izquierda
+- 10: Centro
+- 11: Centro derecha
+
+{{<tag-def-box title="Efecto Karaoke" id="\k">}}
+\\k<i>\<duración></i>
+\\K<i>\<duración></i>
+\\kf<i>\<duración></i>
+\\ko<i>\<duración></i>
+{{</tag-def-box>}}
+
+> _Tenga en cuenta que estas etiquetas por sí solas solo crean algunos efectos muy específicos
+> y todos los demás efectos se crean con una combinación de múltiples diferentes
+> etiquetas._
+
+La familia de etiquetas `\k` marca los subtítulos para efectos de karaoke especificando la duración de cada sílaba. Coloque una etiqueta `\k` antes de cada sílaba en la línea.
+
+La _duración_ se da en centisegundos, ej. una _duración_ de 100 equivale a 1 segundo. Por lo general, uno no ingresa etiquetas `\k` manualmente, sino que utiliza herramientas de sincronización de karaoke como [modo karaoke de Aegisub]({{<relref path="Tutorials#sincronizar-karaoke">}}).
+
+Las diferentes etiquetas `\k` crean varios efectos:
+
+- `\k`: Antes de resaltar, la sílaba se rellena con el color y alfa secundarios. Cuando comienza la sílaba, el relleno cambia instantáneamente para usar color y alfa primarios.
+- `\K` y `\kf`: Estos dos son idénticos. Tenga en cuenta que `\K` es una K mayúscula y es diferente de `\k` minúscula. El relleno de sílaba comienza en color secundario, y cuando comienza la sílaba el relleno cambia de secundario a primario con un barrido de izquierda a derecha, y el barrido termina cuando el tiempo de la sílaba termina.
+- `\ko`: Similar a `\k`, excepto que antes de resaltar, el borde/contorno de la sílaba se elimina y aparece instantáneamente cuando comienza la sílaba.
+
+_Nota: Hay una etiqueta de karaoke adicional, `\kt`, que es muy diferente a las demás. Rara vez es útil y Aegisub no soporta esa etiqueta; por lo tanto no está documentada.
+
+{{<tag-def-box title="Estilo de salto" id="\q">}}\\q<i>\<estilo></i>{{</tag-def-box>}}
+
+Determina cómo se aplica el salto de línea a la línea de subtítulo. Las siguientes \_estilo_s están disponibles:
```

Comment:
```suggestion
Determina cómo se aplica el salto de línea a la línea de subtítulo. Los siguientes \_estilo_s están disponibles:
```
I feel like this is wrong even on the original: \_style_s

Reply:
Agreed, will fix here and so the same for English in #33